src/App.js

```diff
@@ -3,19 +3,26 @@ import ListWrapper from './components/ListWraper/ListWraper';
 import UserData from './data/userData';
 import Form from './components/Form/Form';
 
-
 class App extends Component {
   state = {
     items: UserData(),
   };
+
+  
+  changeState = newState => {
+    this.setState({
+      items: newState
+    })
+  }
```

Comment:
tak nie można Krzysiu,

setSTate to juz jest funkcja która updatuje state,

Reply:
No zgadza się, aktualizuję state App na inny z komponentów Form i Listitems